acpi-dumper/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "tools"
+version = "0.0.0"
+publish = false
+authors = ["Isaac Woods", "Matt Taylor"]
+repository = "https://github.com/rust-osdev/acpi"
+description = "ACPI table dumper"
+categories = ["hardware-support", "no-std"]
```

Comment:
The dumper itself uses the stdlib, so probably shouldn't be in the `no-std` category